backend/src/test/java/moadong/fixture/ClubFixture.java

```diff
@@ -0,0 +1,83 @@
+package moadong.fixture;
+
+import moadong.club.entity.Club;
+import moadong.club.entity.ClubRecruitmentInformation;
+import moadong.club.enums.ClubRecruitmentStatus;
+import moadong.club.enums.ClubState;
+import moadong.club.payload.request.ClubInfoRequest;
+import moadong.club.payload.request.ClubRecruitmentInfoUpdateRequest;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.Map;
+
+public class ClubFixture {
+    public static final ClubState DEFAULT_STATE = ClubState.UNAVAILABLE;
+    public static Club createClub() {
+        return Club.builder()
+                .name("테스트 동아리")
+                .category("학술")
+                .division("프로그래밍")
+                .clubRecruitmentInformation(createRecruitmentInfo())
+                .build();
+    }
```

Comment:
제안과 같이 mock을 활용하면 객체의 수정할 수 없는 부분도 테스트 가능하고, 테스트를 위해 프로덕션 코드가 수정될 가능성이 많이 줄어듭니다!
```suggestion
    public static Club createClub(String clubId, String name) {
        Club club = mock(Club.class);
        when(club.getId()).thenReturn(clubId);
        when(club.getName()).thenReturn(name);
        return club;
    }
```

Reply:
제안대로 수정해보겠습니다